nonprofit-board-management.php

```diff
@@ -59,28 +59,33 @@ public function __construct(){
 		// Run the activation when a new multisite blog is created.
 		add_action( 'wp_initialize_site', array( $this, 'multisite_activate_plugin' ) );
 
-        if( is_admin() ){
-          add_action( 'wp_dashboard_setup', array( $this, 'remove_dashboard_widgets' ) );
+		if ( is_admin() ) {
+			add_action( 'wp_dashboard_setup', array( $this, 'remove_dashboard_widgets' ) );
 
-          //Setup top level menu
-          add_action( 'admin_menu', array( $this, 'create_menu' ), 10 );
+			// Setup top level menu.
+			add_action( 'admin_menu', array( $this, 'create_menu' ), 10 );
 
-          //Load CSS and JS
-          add_action( 'admin_enqueue_scripts', array( $this, 'insert_css') );
-          add_action( 'admin_enqueue_scripts', array( $this, 'insert_js') );
+			// Load CSS and JS.
+			add_action( 'admin_enqueue_scripts', array( $this, 'insert_css') );
+			add_action( 'admin_enqueue_scripts', array( $this, 'insert_js') );
 
-          //Add our board members dashboard widget
-          add_action('wp_dashboard_setup', array( $this, 'add_board_members_dashboard_widget' ) );
+			// Add our board members dashboard widget.
+			add_action( 'wp_dashboard_setup', array( $this, 'add_board_members_dashboard_widget' ) );
 
-          //Remove the help tabs for all board members
-          add_action( 'in_admin_header', array( $this, 'remove_help_tabs' ) );
+			// Remove the help tabs for all board members.
+			add_action( 'in_admin_header', array( $this, 'remove_help_tabs' ) );
 
-          //Add notice to admin who can't serve on board in case they want to.
-          add_action( 'admin_notices', array( $this, 'show_admins_notices' ) );
+			// Add notice to admin who can't serve on board in case they want to.
+			add_action( 'admin_notices', array( $this, 'show_admins_notices' ) );
 
-          //Allow admin to click a button and start serving on the board.
-          add_action( 'wp_ajax_allow_user_to_serve', array( $this, 'allow_user_to_serve' ) );
-        }
+			// Allow admin to click a button and start serving on the board.
+			add_action( 'wp_ajax_allow_user_to_serve', array( $this, 'allow_user_to_serve' ) );
+
+			// Used to check if a multisite super admin specifically has the 'serve_on_board' capability.
+			if ( is_multisite() ) {
```

Comment:
I found a bug in multisites that prevented a super admin from adding themselves as a board member because current_user_can() always returns true for super admins unless explicitly set otherwise.

Reply:
Great work on this!